ra2ce/analysis/adaptation/adaptation.py

```diff
@@ -119,8 +121,13 @@ def calculate_bc_ratio(
             cost_gdf (GeoDataFrame): Gdf containing the cost of the adaptation options.
 
         Returns:
-            GeoDataFrame: Gdf containing the benefit-cost ratio of the adaptation options.
+            GeoDataFrame: Gdf containing the benefit-cost ratio of the adaptation options,
+                including the relevant attributes from the original graph (geometry).
         """
+        # Copy the relevant attributes from the original graph
+        _orig_gdf = self.graph_file_hazard.get_graph()
+        benefit_gdf["geometry"] = _orig_gdf.geometry
```

Comment:
are we exporting this as well? As far as I know we deliberately remove the column `geometry` when exporting `gdf` files (check `AnalysisResultWrapperExporter`).

Reply:
We remove the column geometry only in case of csv export